MatomoTracker/MatomoUserDefaults.swift

```diff
@@ -99,6 +99,18 @@ internal struct MatomoUserDefaults {
             userDefaults.set(newValue, forKey: MatomoUserDefaults.Key.lastOrder)
         }
     }
+    
+    func reset() {
```

Comment:
I was thinking about the term `reset` here. I was wondering, since this is a wrapper of `UserDefaults` if we want to keep the name close to the names of `UserDefaults`. So something like `removeAllObjects` or so. What do you think?